### DISCUSSION TITLE
action center

### DISCUSSION THREAD

**Discussion by Hottihotte** [valinet/ExplorerPatcher#3903] [General]
Is there a way to restore the Action Center from Windows 10?
So not the weirdly split window from Windows 11.
Windows 11
![win11](https://github.com/user-attachments/assets/2b901d82-5faf-44b6-8bc5-46d15ec793c3)

Windows 10
![win10](https://github.com/user-attachments/assets/b846e7c3-9542-49d6-80ee-21af8686ca59)

And take the icon for active applications from Windows 10, because I liked that better than the current one from "Windows 10 ExplorerPatcher"

Windows 10 ExplorerPatcher
![Anwendung11](https://github.com/user-attachments/assets/491da46a-bbca-4ab8-ae1b-ea16be326cc1)

Windows 10
![Anwendung10](https://github.com/user-attachments/assets/245b3e44-3459-43d1-8685-07b177113f9f)

**Comment by pyrates999** [comment]
Only if the code for it still exists in windows 11 24h2.

**Comment by milestprower92** [comment]
To restore the Windows 10 action center on 11
1. Open the Registry editor
2. go to this path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion
3. create a new key and name it Control Center
4. now in that key create a new DWORD value and name it UseLiteLayout and set its value to 1
5. restart your computer

and the taskview icon has been reverted to the one from Windows 10 1507-1709 on the reimplemented taskbar because windows 11's taskview looks like the one from those early versions of Windows 10

**Reply by Hottihotte** [reply]
> To restore the Windows 10 action center on 11
> 
>     1. Open the Registry editor
> 
>     2. go to this path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion
> 
>     3. create a new key and name it Control Center
> 
>     4. now in that key create a new DWORD value and name it UseLiteLayout and set its value to 1
> 
>     5. restart your computer

This works but there is one disadvantage.
The font color in the Start and Action Center changes.
before (normal)
![1](https://github.com/user-attachments/assets/1cbe16c0-cad3-4455-ad0e-c8830aea1f27)

and then
![2](https://github.com/user-attachments/assets/33eb60f0-c919-4ca3-a066-93293f028c9f)

The reason for this is probably under Settings-Personalization-Colors in Windows 10
Show accent color on the following areas
Start, taskbar and info center

In Windows 11, however, there is only
Show accent color on start and taskbar
You could change the registry entry
`Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize
"AppsUseLightTheme"=dword:00000001`
set to 0 and the display is correct again but then all Explorer windows, task manager or apps (like browser) would be black

**Reply by milestprower92** [reply]
> > To restore the Windows 10 action center on 11
> > 
> >     1. Open the Registry editor
> > 
> >     2. go to this path: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion
> > 
> >     3. create a new key and name it Control Center
> > 
> >     4. now in that key create a new DWORD value and name it UseLiteLayout and set its value to 1
> > 
> >     5. restart your computer
> 
> This works but there is one disadvantage.
> The font color in the Start and Action Center changes.
> before (normal)
> ![1](https://github.com/user-attachments/assets/1cbe16c0-cad3-4455-ad0e-c8830aea1f27)
> 
> and then
> ![2](https://github.com/user-attachments/assets/33eb60f0-c919-4ca3-a066-93293f028c9f)
> 
> The reason for this is probably under Settings-Personalization-Colors in Windows 10
> Show accent color on the following areas
> Start, taskbar and info center
> 
> In Windows 11, however, there is only
> Show accent color on start and taskbar
> You could change the registry entry
> `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize
> "AppsUseLightTheme"=dword:00000001`
> set to 0 and the display is correct again but then all Explorer windows, task manager or apps (like browser) would be black

That's a known issue with the UseLiteLayout dword and there are no plans to fix and add the 10 action center as an option on ExplorerPatcher

**Reply by Hottihotte** [reply]
> That's a known issue with the UseLiteLayout dword and there are no plans to fix and add the 10 action center as an option on ExplorerPatcher

So the only thing left is to accept the new?
